clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -2619,6 +2619,9 @@ private ConfigEntry.ConfigSource configSource(DescribeConfigsResponse.ConfigSour
             case DYNAMIC_BROKER_LOGGER_CONFIG:
                 configSource = ConfigEntry.ConfigSource.DYNAMIC_BROKER_LOGGER_CONFIG;
                 break;
+            case CLIENT_METRICS_CONFIG:
```

Comment:
Hmm, this method seems very specific to get/create topic. Do we really need to add client metric here?

Reply:
You are right, this should not be needed. I have removed it, I am not sure why other config handlers are required here when the method is just used by topics code. May be previously it was used elsewhere as well.

Reply:
Some of the configs could be set at both topic and broker levels. So, there is a hierarchy.

Reply:
Sounds good, thanks for letting me know.